app/frontend/src/internationalization/trans/chefs/fr/fr.json

```diff
@@ -1083,5 +1108,89 @@
       "history": "HISTOIRE",
       "user": "Utilisateur"
     }
+  },
+  "eventStreamConfig": {
+    "info": "Configurez le flux d’événements pour votre formulaire.",
+    "create": "Créer une nouvelle configuration de flux d&#39;événements",
+    "createError": "La configuration du flux d&#39;événements n&#39;a pas pu être créée.",
+    "createSuccess": "Configuration du flux d&#39;événements créée avec succès.",
+    "createTitle": "Nouvelle configuration du flux d&#39;événements",
+    "delete": "Supprimer",
+    "deleteSuccess": "La configuration du flux d&#39;événements a été supprimée avec succès.",
+    "deleteError": "Une erreur s&#39;est produite lors de la suppression de la configuration du flux d&#39;événements.",
+    "edit": "Modifier",
+    "editError": "La configuration du flux d&#39;événements n&#39;a pas pu être mise à jour.",
+    "editSuccess": "La configuration du flux d&#39;événements a été mise à jour avec succès.",
+    "editTitle": "Modifier la configuration du flux d&#39;événements",
+    "fetchError": "Une erreur s&#39;est produite lors de la récupération de la configuration du flux d&#39;événements.",
+    "save": "Sauvegarder"
+  },
+  "encryptionKey": {
+    "info": "Configurez les clés de chiffrement pour votre formulaire.",
+    "create": "Créer une nouvelle clé de chiffrement",
+    "createError": "La clé de chiffrement n&#39;a pas pu être créée.",
+    "createSuccess": "Clé de cryptage créée avec succès.",
+    "createTitle": "Nouvelle configuration de clé de chiffrement",
+    "delete": "Supprimer",
+    "deleteSuccess": "La clé de cryptage a été supprimée avec succès.",
+    "deleteError": "Une erreur s&#39;est produite lors de la suppression de la clé de chiffrement.",
+    "edit": "Modifier",
+    "editError": "La clé de chiffrement n&#39;a pas pu être mise à jour.",
+    "editSuccess": "Clé de cryptage mise à jour avec succès.",
+    "editTitle": "Modifier la clé de cryptage",
+    "fetchError": "Une erreur s&#39;est produite lors de la récupération de la clé de chiffrement.",
+    "fetchListError": "Une erreur s&#39;est produite lors de la récupération de la liste des clés de chiffrement.",
+    "save": "Sauvegarder"
+  },
+  "externalAPI": {
+    "info": "Configurez les API externes à utiliser dans votre formulaire.",
+    "create": "Créer une nouvelle API externe.",
+    "createError": "L&#39;API externe n&#39;a pas pu être créée.",
+    "createSuccess": "API externe créée avec succès.",
+    "createTitle": "Nouvelle configuration de l&#39;API externe",
```

Comment:
For some reason accented characters are being encoded in two different ways. One way isn't working.
![image](https://github.com/user-attachments/assets/63c7b2d3-28b7-47d9-8781-c8dd7239bde2)